io-backend/src/main/java/agh/io/iobackend/model/map/MapStructure.java

```diff
@@ -0,0 +1,19 @@
+package agh.io.iobackend.model.map;
+
+import agh.io.iobackend.model.Vector;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.ArrayList;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class MapStructure {
```

Comment:
Rozumiem że mapa jest w postaci 3 zbiorów, przechowujących współrzędne danych obiektów. Czy jest to jakoś przemyślane? Bo bardziej intuicyjne wydaje się mieć macierz-mapę, z elementami reprezentującymi typ pola.

Reply:
tak, jest to dostosowane do tego tego w jaki sposób jest to na frontendzie zaimplementowane; ale w sumie możliwe że zapomniałam dodać `size` - dla szerokości i wysokości; trochę zaoszczędzimy też pamięci, nie musząc w ten sposób przechowywać pustych pól jakoś zakodowanych

Reply:
Tak sądziłem że to nam narzuca Front. To co robimy z tym `size`, dodasz szybko?